src/Writing/Writer.php

```diff
@@ -142,7 +142,7 @@ public function generateOpenAPISpec(array $groupedEndpoints): string
         /** @var OpenAPISpecWriter $writer */
         $writer = app()->makeWith(OpenAPISpecWriter::class, ['config' => $this->config]);
         $spec = $writer->generateSpecContent($groupedEndpoints);
-        return Yaml::dump($spec, 20, 2, Yaml::DUMP_EMPTY_ARRAY_AS_SEQUENCE | Yaml::DUMP_OBJECT_AS_MAP);
+        return Yaml::dump($spec, 20, 2, Yaml::DUMP_EMPTY_ARRAY_AS_SEQUENCE | Yaml::DUMP_OBJECT_AS_MAP | Yaml::DUMP_NUMERIC_KEY_AS_STRING);
```

Comment:
Hmm, are we sure this doesn't break anything else? I'd much rather go the route of ensuring that the response status codes are actually strings, so they get dumped as strings without needing this.